azure-pipelines/templates/steps/maven-release/maven-release-jobs.yml

```diff
@@ -163,16 +162,17 @@ jobs:
     displayName: Download Python Requests Library
     inputs:
       script: pip install requests
-  - task: DownloadSecureFile@1
-    name: credentials
-    displayName: Download Nexus Sonatype Login Credentials
-    inputs:
-      secureFile: NugetADDSonatypeCredentials.json
-      retryCount: 5
-  - task: PythonScript@0
-    displayName: Upload to Nexus Sonatype Staging Directory
+  - task: EsrpRelease@4
     inputs:
-      arguments: ${{ parameters.project }} ${{ parameters.projectVersion }}
-      scriptSource: filePath
-      scriptPath: azure-pipelines/templates/steps/maven-release/upload-to-nexus-sonatype.py
+      ConnectedServiceName: 'AndroidAuth_ESRP_Release'
+      Intent: 'PackageDistribution'
+      ContentType: 'Maven'
+      ContentSource: 'Folder'
+      FolderLocation: '$(Build.ArtifactStagingDirectory)'
+      WaitForReleaseCompletion: true
+      Owners: '$(releaseOwners)'
+      Approvers: '$(releaseApprovers)'
+      ServiceEndpointUrl: 'https://api.esrp.microsoft.com'
+      MainPublisher: 'AndroidAuthSDK'
+      DomainTenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47'
```

Comment:
is it ok for this tenantId to be checked in?

Reply:
I think I should move all these as variables in pipeline.

Reply:
done.